test/verify/check-host-switching

```diff
@@ -258,6 +261,10 @@ class TestHostSwitching(MachineCase, HostSwitcherHelpers):
         b.click("#nav-hosts .nav-item a[href='/@10.111.113.3']")
         b.wait_js_cond('window.location.pathname.indexOf("/@10.111.113.3") === 0')
 
+        # remote hosts should not have the dashboard, since #14590
+        if m.image != "rhel-8-3-distropkg": # Introduced in #14567
+            b.wait_not_in_text("#nav-system", "Dashboard")
```

Comment:
This special case isn't actually necessary any more, but after this lands I'll rebase #14608 and clean it up.